cohort_2/modules/how_to_solve_any_problem.md

```diff
@@ -7,7 +7,7 @@
 3. define MVP
 	1. 	abstract the problem
```

Comment:
Yup, but I think fixing this specific spacing problem may be outside the scope of this PR.